src/Computation/ConstantUnsigned.cs

```diff
@@ -14,7 +14,12 @@ private ConstantUnsigned(Bits size, BigInteger value)
             Integer = value;
         }
 
-        public override BigInteger Integer { get; }
+        public BigInteger Integer { get; }
```

Comment:
Do you still need the `Integer` property if the interface doesn't require it in this form any longer?

Reply:
It was actually needed due to a small detail related to `ArithmeticRIghtShift`, however I have now added it to the `IConstantValue` interface, so it is needed for that now anyway.